migration/esr128.md

```diff
@@ -0,0 +1,575 @@
+    * text/x-ms-rqy
+    * text/x-ms-contact
+    * text/json
+    * text/javascript
+    * text/javascript1.0
+    * text/javascript1.1
+    * text/javascript1.2
+    * text/javascript1.3
+    * text/javascript1.4
+    * text/ecmascript1.5
+    * text/jscript
+    * text/livescript
+    * text/x-ecmascript
+    * text/x-javascript
+  * これらのMIME Type以外のテキスト型データを直接外部アプリケーションで開くことはできなくなりました。
+    （`Handlers` ポリシーでも制御はできません。）
+  * これらのMIME Type以外のテキスト型データを自動的にダウンロードさせるには、サーバー側で`Content-Disposition: attachment`ヘッダーを設定する、`Content-Type: application/octet-stream`で送出するようにするか、もしくはそのようにレスポンスヘッダーを書き換えるアドオンを使う必要があります。
+
+
+## インストール・更新
+
+* Chromeからのデータ移行時に、拡張機能も移行するようになった。（Firefox 119）
+
+
+
+## Firefox View
+
+![（画像：Firefox Viewのスクリーンショット）](esr128/firefox-view.png)
+
+* 「開いているすべてのタブ」「他のデバイスのすべてのタブ」の一覧表示に対応した。（Firefox 119）  
+  ![（画像：Firefox Viewでの履歴項目の並べ替え用UIのスクリーンショット）](esr128/firefox-view-open-tabs.png)
+  * 開かれているタブの一覧の表示順を「最近使用した順」（既定値）と「タブバー上の並び順」の間で切り替えられるようになった。（Firefox 124）
+* 履歴の並べ替え表示に対応した。（Firefox 119）  
+  ![（画像：Firefox Viewでの履歴項目の並べ替え用UIのスクリーンショット）](esr128/firefox-view-sort-history.png)
+* 各セクションの内容を横断した検索に対応した。（Firefox 123）
+* 開かれているタブの一覧において、ピン留めされたタブも表示するようになった。これにより、メディア再生中のタブのミュート状態の切り替え、ブックマークの作成などの操作をより簡単に行えるようになった。（Firefox 125）  
+  ![（画像：Firefox Viewでピン留めされたタブが表示されている様子のスクリーンショット）](esr128/firefox-view-open-pinned-tabs.png)
+* ポリシー設定でFirefox Accountが無効化されている場合は、Firefox Viewの画面内に「他のデバイスのすべてのタブ」が表示されないようになった。（Firefox 126）
+
+
+## 内蔵PDFビューワー
+
+![（画像：PDFビューワーのスクリーンショット）](esr128/pdfjs.png)
+
+* 既存のテキスト形式の注釈を編集できるようになった。（Firefox 116）  
+  ![（画像：PDFビューワーでテキスト形式の注釈を編集している様子のスクリーンショット）](esr128/pdfjs-add-text.png)
+* 画像の追加や代替テキストの設定をできるようになった。（Firefox 119）  
+  ![（画像：PDFビューワーで画像を追加している様子のスクリーンショット）](esr128/pdfjs-add-image.png)
+  * この機能は、`Preferences` ポリシーで `pdfjs.enableStampEditor` を `false` にすることで無効化できます。
+* PDFに追加された線、テキスト、画像などを削除するためのフローティングボタンを追加。（Firefox 121）
+<!--
+  * この機能は、`Preferences` ポリシーで `pdfjs.enableHighlightFloatingButton` を `false` にすることで無効化できます。
+    →実際に試してみたところ、設定の状態に関わらずこれらのボタンは常に有効だった。
+-->
+* F7キーによるキャレットブラウジングがPDFビューワー上でも使用可能になった。（Firefox 124）
+* 選択したテキストをマーカー風に強調表示できるようになった。（Firefox 125）  
+  ![（画像：PDFビューワーでテキストを強調表示している様子のスクリーンショット）](esr128/pdfjs-highlight-text.png)
+  * この機能は、`Preferences` ポリシーで `pdfjs.enableHighlightEditor` を `false` にすることで無効化できます。
+
+
+## 動画・音声・音楽再生
+
+* 動画の別ウィンドウ再生（Picture-in-Picture）機能において、音量を調節できるようになった。（Firefox 116）
+<!--単なる不具合修正のため省略
+* スクリーンリーダー使用時にYouTubeの動画リストを正しくスクロールできるようになった。（Firefox 117）
+* Google Meetにおいて、背景のぼかしを適用できるようになった。（Firefox 118）
+-->
+* 動画などのメディアファイルを `application/octet-stream` としてダウンロードする際に、ファイルの再生を試行せずにダウンロードするようになった。（Firefox 119）
+* WindowsとLinuxにおいて、Picture-in-PictureのウィンドウをCtrlキーを押しながらドラッグすることで、ウィンドウが画面の最も近い角にスナップされるようになった。（Firefox 120）
+* Windowsにおいて、[AV1 Video Extension](https://apps.microsoft.com/detail/9mvzqvxjbq9v?hl=ja-jp&gl=JP)がインストールされていない場合に、AV1ビデオコーデックのハードウェアアクセラレーションを有効化するために`about:support`の画面でAV1 Video Extensionのインストールを促すようになった。（Firefox 121）
```

Comment:
```suggestion
* Windowsにおいて、[AV1 Video Extension](https://apps.microsoft.com/detail/9mvzqvxjbq9v?hl=ja-jp&gl=JP)がインストールされていない場合に、AV1ビデオコーデックのハードウェアアクセラレーションを有効化するために`about:support`の画面でAV1 Video Extensionのインストールを促すようになった。（Firefox 121）
  ![（画像：AV1 Video Extension「拡張機能をインストール」リンクのスクリーンショット）](esr128/av1-video-codec-installation-link.png)
```
It would be better to show the change with a screenshot.